common/djangoapps/util/sysadmin_utils.py

```diff
@@ -0,0 +1,38 @@
+"""
+Utilities for edx-sysadmin plugin.
+"""
+
+from django.conf import settings
+
+from edx_django_utils.plugins import get_plugin_apps
+from openedx.core.djangoapps.plugins.constants import ProjectType
+
+
+def is_edx_sysadmin_installed():
+    """
+    Checks if edx-sysadmin plugin is installed in the platform
+    :return boolean: True if plugin is installed else False
+    """
+    for plugin in get_plugin_apps(ProjectType.LMS):
```

Comment:
I think [PluginManager's](https://github.com/edx/edx-django-utils/blob/71f239156bbbaa2a5841eba55029433b14952b90/edx_django_utils/plugins/plugin_manager.py#L42) `get_plugin` method is better way of getting an installed plugin

Reply:
Converted to that.